simcon_project/users/views/researcher_settings_page.py

```diff
@@ -0,0 +1,7 @@
+from django.shortcuts import render
+from django.contrib.auth.decorators import login_required
+
+
+@login_required
```

Comment:
I tried using this decorator how you have it, and it didn't seem to guard anything. Does it work for you?
The syntax that worked for me was: `@login_required('/accounts/login')`
I'd like to use a global variable for the path since hardcoding a login path isn't great. Haven't figured out how to do that yet though.

Reply:
seemed to guard! but i can add that in